docs/reference/Core/NotificationCenter/NotificationListener/index.html

```diff
@@ -44,7 +44,7 @@
 <div class=slotDescription>
 Checks to see if aNotification matches the listener criteria and if so, sends it to the target. Implementation:
 <pre>
-if((name == nil or name == n name) and(object == nil or sender == n sender),
+if((name == nil or name == n name) and(sender == nil or sender == n sender),
```

Comment:
This change looks correct to me, it matches the logic change from commit d86ab173923128676d482ddc318f7df9cf342e23.